cli/test/cli.ts

```diff
@@ -497,4 +502,63 @@ describe("CLI", () => {
                 done();
             });
     });
+
+    it("release doesn't allow releasing .zip file", (done: MochaDone): void => {
+        var command: cli.IReleaseCommand = {
+            type: cli.CommandType.release,
+            appName: "a",
+            deploymentName: "Staging",
+            description: "test releasing zip file",
+            mandatory: false,
+            appStoreVersion: "1.0.0",
+            package: "/fake/path/test/file.zip"
+        };
+
+        var release: Sinon.SinonSpy = sandbox.spy(cmdexec.sdk, "release");
```

Comment:
You can move this inside releaseHelperFunction(...).

Same for subsequent tests.